awx/ui/client/src/templates/prompt/prompt.block.less

```diff
@@ -1,12 +1,13 @@
 .Prompt .modal-dialog {
-    width: 700px;
+    max-width: 700px;
```

Comment:
There is a chance with this that a modal with very little content would appear too narrow (i.e. much less than 700px wide).

Consider something like this instead. A `max-width` of 100vh will prevent the modal from being wider than the viewport:

```
width: 700px;
max-width: 100vh;
```

Reply:
Good idea. Thanks